src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -318,13 +318,13 @@ private String returnTypeComment(Method method, MethodConfig config) {
         resourceTypeName = "a " + jsTypeName(resourceType);
       }
       return callbackMessage
-          + "\n@returns {Stream|gax.EventEmitter}\n"
+          + "\n@returns {Stream|Promise}\n"
           + "  An object stream which emits "
           + resourceTypeName
           + " on 'data' event.\n"
           + "  When the callback is specified or streaming is suppressed through options,\n"
-          + "  it will return an event emitter to handle the call status and the callback\n"
-          + "  will be called with the response object.";
+          + "  it will return a promise that resolves to the response object. The promise\n"
+          + "  has cancel method which cancels the ongoing API call.";
```

Comment:
Here I wanted to say the promise has a method, and the name of the method is "cancel", and 'a cancel method' sounds a bit unclear to me about the method name.

Changed as `has a method named "cancel" which...`